docker/development.yml

```diff
@@ -90,6 +90,10 @@ services:
       context: ../src/api/parser
       cache_from:
         - docker.cdot.systems/parser:buildcache
+    environment:
+      # In development and testing, the SSO service needs to contact the Supabase
+      # service directly via Docker vs through the http://localhost/v1/supabase domain.
+      - SUPABASE_URL=https://dev.supabase.telescope.cdot.systems/
```

Comment:
NOTE: this is the URL for the console, not the api.  Use `https://dev.api.telescope.cdot.systems/v1/supabase`